packages/ruby-2.7/packaging

```diff
@@ -15,8 +15,14 @@ if [ $(uname -s) == "Darwin" ]; then
 fi
 
 # Use Clang if available; the resulting Ruby is faster
-if [ -x /usr/bin/clang ]; then
-  export CXX=/usr/bin/clang++ CC=/usr/bin/clang
+if command -v clang &> /dev/null
+then
+	export CC="$(command -v clang)"
```

Comment:
Using `command -v` works on any bash-compliant shell. It is a builtin. The most precise solution would be one that is robust to aliases, such as `which --skip-alias clang` but this assumes `which` to be installed in a sufficiently new version.